models/domainattention/domain_attention_models.py

```diff
@@ -85,7 +85,10 @@ def call(self, inputs, training=None, mask=None):
             x = tf.expand_dims(tf.transpose(weights), axis=2) * feature_vectors
 
         # sum over weighted vectors so it will be a weighted mean
-        x = tf.reduce_sum(x, axis=0)
+#         x = tf.reduce_sum(x, axis=0)
```

Comment:
I think this should go into else block. When we do not use element-wise, it is correct to sum the features